calculator.rb

```diff
@@ -0,0 +1,105 @@
+#checks operator input 
+def operator (user_operation)
+  case user_operation
+    when "add", "+"
+      print "Plese enter your first number: "
+      user_num1 = gets.chomp
+      user_num1 = num_check (user_num1)
+
+      print "Please enter your second number: "
+      user_num2 = gets.chomp
+      user_num2 = num_check(user_num2)
+      
+      addition(user_num1, user_num2)
+
+    when "subtract", "-"
+      print "Plese enter your first number: "
+      user_num1 = gets.chomp
+      user_num1 = num_check (user_num1)
+
+      print "Please enter your second number: "
+      user_num2 = gets.chomp
+      user_num2 = num_check(user_num2)
+
+      subtraction(user_num1, user_num2)
+
+    when "multiply", "*"
+      print "Plese enter your first number: "
+      user_num1 = gets.chomp
+      user_num1 = num_check (user_num1)
+
+      print "Please enter your second number: "
+      user_num2 = gets.chomp
+      user_num2 = num_check(user_num2)
+
+      multiplication(user_num1, user_num2)
+
+    when "divide", "/"
+      print "Plese enter your first number: "
+      user_num1 = gets.chomp
+      user_num1 = num_check (user_num1)
+
+      print "Please enter your second number: "
+      user_num2 = gets.chomp
+      user_num2 = num_check(user_num2)
+
+      division(user_num1, user_num2)
+
+    else 
+      puts "NOT AN OPERATOR!"
+      exit
```

Comment:
This is kind of a frustrating user experience. Could you have used a loop to improve it?